.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -40,129 +40,23 @@ body:
           If this isn't a bug and you have a question or support request, please email user@apache.beam.com with a description of the problem instead of opening this issue.
     validations:
       required: true
-  - type: dropdown
-    id: priority
-    attributes:
-      label: Issue Priority
-      description: What priority is this bug? See https://beam.apache.org/contribute/issue-priorities for the meaning and expectations around issue priorities.
-      options:
-        - "Priority: 3"
-        - "Priority: 2"
-        - "Priority: 1"
-        - "Priority: 0"
```

Comment:
Do we want to get rid of priority? I find this pretty helpful when I file issues myself, and I imagine this will be a friendlier experience for non-committers who often file issues

Reply:
Mostly I think that non-committers do not have context for what a priority should be, and adding a label is pretty much as easy as doing it as part of the template. But I'm OK to add it back.

Reply:
PTAL. I did change some of the options, and added explanations in parens. I kept the syntax identical, assuming the automation is expecting it.